doc/iscsi_fw_login.8

```diff
@@ -4,9 +4,9 @@ iscsi_fw_login \- Login to all iSCSI Firmware targets
 .SH SYNOPSIS
 .B iscsi_fw_login
 .SH DESCRIPTION
-This helper function logs into all known iSCSI firmware targets.
+Logs into all known iSCSI firmware targets.
 .P
-It is meant as a helper function to be called by udev when new
+Meant as an udev hook to be called by udev when new
```

Comment:
I believe that should be "Meant as a udev hook ...".

Reply:
It's /ɤdəv/ in my accent, so an; you might americanise this to /judəv/ i guess, which would be a?

Reply:
It's a nit, but the correct grammar in english is to use "a".